src/formConfig.ts

```diff
@@ -0,0 +1,85 @@
+export type Auction = {
+  auctioningToken: string
+  biddingToken: string
+  sellAmount: string
+  minBuyAmount: string
+  minFundingThreshold?: string
+  orderCancellationEndDate?: string
+  auctionEndDate?: string
+  minBuyAmountPerOrder?: string
+  isAtomicClosureAllowed?: boolean
+  allowListManager?: string
+  allowListData?: string
+}
+
+export type FormKeys = keyof Auction
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const DEFAULT_FORM_PARAMS: Readonly<Record<FormKeys, any>> = {
+  auctioningToken: '',
+  biddingToken: '',
+  sellAmount: '0',
+  minBuyAmount: '0',
+  minFundingThreshold: '0',
+  orderCancellationEndDate: '',
+  auctionEndDate: '',
+  minBuyAmountPerOrder: '0.01',
+  isAtomicClosureAllowed: true,
```

Comment:
Could we please set this value to false?

Reply:
Done